docs/content/docs/KnownIssues.md

```diff
@@ -11,3 +11,14 @@ type: docs
 Disable **Settings > Experimental > Only sync the active variant**
 
 https://stackoverflow.com/questions/54201216/android-studio-3-3-new-gradle-sync-is-not-supported-due-to-containing-kotlin-mo/
+
+## Tests run from Android Studio are failed in runtime with `java.lang.NoClassDefFoundError`
+
+The problem occurs only when running a test from Android Studio because of wrongly generated Run Configuration.
+Wherein test run from command line via `./gradlew test` ends up normally.  
+Mostly this reproduced in tests that use Kotlin extensions from Test Fixtures from other Gradle modules.
+
+Workaround:
```

Comment:
This is a bug in IDE. Can you please report it?

Reply:
I mean, when you changed settings but the run configuration is not updated accordingly

Reply:
https://youtrack.jetbrains.com/issue/IDEA-235085